source/adapters/level_zero/event.hpp

```diff
@@ -222,6 +223,11 @@ struct ur_event_handle_t_ : _ur_object {
 
   // Get the host-visible event or create one and enqueue its signal.
   ur_result_t getOrCreateHostVisibleEvent(ze_event_handle_t &HostVisibleEvent);
+
+  // Keeps track of whether we are using Counter-based Events.
+  bool CounterBasedEventsEnabled = false;
+
+  bool usingCounterBasedEvents() const { return CounterBasedEventsEnabled; }
```

Comment:
`CounterBasedEventsEnabled` looks like a public struct member, what's the benefit of defining a getter function to return it?